### DISCUSSION TITLE
Length condition

### DISCUSSION THREAD

**Discussion by AoElite** [OpnForm/OpnForm#48] [Ideas & Feature Requests]
I think a length condition would be a great addition to the logic part of the builder.

**Comment by JhumanJ** [comment]
Hey @AoElite, how would you see this? If (answer length of field x) > x then do something? May I ask for an example use-case?

**Reply by AoElite** [reply]
So it essentially just returns the message length and it would have <>= operators & a number. Basically exactly how you described.

**Reply by JhumanJ** [reply]
We've added this to our roadmap :)
